projects/ion/src/lib/divider/divider.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input } from '@angular/core';
+
+type Orientation = 'horizontal' | 'vertical';
+
+type Type = 'solid' | 'dashed' | 'text';
+
+export interface IonDividerProps {
+  label?: string;
+  orientation?: Orientation;
+  type?: Type;
+}
+
+@Component({
+  selector: 'ion-divider',
+  templateUrl: './divider.component.html',
+  styleUrls: ['./divider.component.scss'],
+})
+export class DividerComponent {
+  @Input() label?: string = '';
```

Comment:
```suggestion
  @Input() label = '';
```
In this situation, type can be inferred by typescript, you don't need to explicitly add it (more about it [here](https://www.typescriptlang.org/docs/handbook/type-inference.html)).